.env.example

```diff
@@ -5,4 +5,4 @@ SEC_SERVER_PORT=3000
 # Database
 DB_HOST=127.0.0.1
 DB_PORT=27017
-DB_NAME=basehub
+DB_NAME=admin
```

Comment:
Is there a reason why the name keeps changing, why not just leave it at nothing?